servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/TimeoutFromRequest.java

```diff
@@ -29,7 +29,7 @@
 /**
  * A function to determine the appropriate timeout to be used for a given {@link HttpRequestMetaData HTTP request}.
  * The result is a {@link Duration} which may be null if no timeout is to be applied. If the function blocks then
- * {@link #influenceStrategy(HttpExecutionStrategy)} should alter the execution strategy as required.
+ * {@link #requiredOffloads()} should specify the execution strategy as required.
```

Comment:
This interface is deprecated, users should migrate from using it instead of migrating from `influenceStrategy` to `requiredOffloads()`. Consider keeping as-is.

Reply:
This was a JavaDocs change that was required when I was contemplating that this change might require removal of `influenceStrategy`. As you suggest it is not critical, but it does avoid a potential problem depending upon the order we remove this interface and `influenceStrategy`